### PR TITLE
Prevent event default behavior (scrolling) on arrow key press.

### PR DESCRIPTION
**Pull request by bicknellr**
Fixes PolymerElements/paper-menu#79.

### CODE REVIEW

**Comment by notwaldorf** [comment]
I think this makes sense! LGTM

File: iron-menu-behavior.html [hunk 1]
```diff
@@ -266,6 +266,7 @@
     _onUpKey: function(event) {
       // up and down arrows moves the focus
       this._focusPrevious();
+      event.preventDefault();
```
Comment (notwaldorf) [inline iron-menu-behavior.html]: Do you want `event.stopPropagation()` instead, maybe?

**Comment by notwaldorf** [comment]
Also, pls consider https://github.com/PolymerElements/iron-menu-behavior/pull/30/files, and if that needs to be merged as well

**Comment by bicknellr** [comment]
Whoa, thanks; this and #30 might both be able to be closed just by setting `stop-keyboard-event-propagation`.

**Comment by bicknellr** [comment]
Gah, this is already there...

**Comment by bicknellr** [comment]
Since this needs to call `preventDefault` rather than `stopPropagation` the `stop-keyboard-event-propagation` flag isn't going to work, so I just called `preventDefault` in the relevant arrow key cases.

**Comment by notwaldorf** [comment]
LGTM

**Pull request merged**